modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/DefaultSourceCallback.java

```diff
@@ -238,4 +238,20 @@ public TransactionConfig getTransactionConfig() {
   public SourceConnectionManager getSourceConnectionManager() {
     return messageSource.getSourceConnectionManager();
   }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String getOwningSourceName() {
```

Comment:
why don't you just have one method which returns the SourceModel?

Reply:
just to not expose everything